examples/jsm/controls/OrbitControls.js

```diff
@@ -358,6 +359,27 @@ class OrbitControls extends EventDispatcher {
 		const pointers = [];
 		const pointerPositions = {};
 
+
+		function getPosition( obj, vec ) {
+
+			if ( ! obj.parent || obj.parent.type === 'Scene' )
+				return vec.copy( obj.position );
+
+			obj.getWorldPosition( vec );
+
+			return vec;
+
+		}
+
+		function setPosition( obj, val ) {
+
+			if ( obj.parent )
+				obj.parent.worldToLocal( val );
+
+			obj.position.copy( val );
+
+		}
```

Comment:
Instead of having `setPosition()`, can't we just always use `scope.object.worldToLocal()`?